app/views/facility_notifications/index.html.haml

```diff
@@ -10,7 +10,22 @@
   = render "shared/transactions/top", tab: "notifications"
 
 - if @order_details.any?
-  = render "shared/transactions/table", order_details: @order_details
+  #table_billing
+    = form_tag url_for(controller: params[:controller], action: :send_notifications), id: "journals_create_form", name: "journals_create_form", method: :post, class: "form-horizontal" do
```

Comment:
Now that this is pulled up and only being used in one place, can we make it less generic? I'm thinking use `facility_notififications_send_notifications_path` rather than `url_for`. 

Also, can we change the name and id to something that fits the context (or remove them altogether)? If they're being used for CSS/JS targeting, I'll accept that it might be too much effort to be worth fixing, but if it's straightforward can we clean it up?

Reply:
Good call, I don't think we need the id/name in this context.